sormas-api/src/test/java/de/symeda/sormas/api/utils/DefaultUserHelperTest.java

```diff
@@ -0,0 +1,153 @@
+package de.symeda.sormas.api.utils;
+
+import de.symeda.sormas.api.user.UserDto;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class DefaultUserHelperTest {
+
+    private static final String DEFAULT_ADMIN_USERNAME = "admin";
```

Comment:
I wouldn't hard cod the default names here again. We have a centralized place for it, so we can IMHO re-use the constants provided by `DefaultUserHelper` in the related tests.

Reply:
wdyt @schwzr @Snorzze

Reply:
My thoughts: What happens when something changes accidentally at the central place? (e.g. typo when refactoring code). So we know these default logins for sure and so we should ensure, that the "right" default usernames and passwords are uses. Otherwise we would test the Helper with the data it self provides and thus, would not recognize a possible misspelling

Reply:
Sounds valid. Might be a review point by the SORMAS guys, though :)
Both variants are OK for me.